### DISCUSSION TITLE
Restricted access after changing table name on postgres

### DISCUSSION THREAD

**Discussion by oktasense** [directus/directus#6803] [Archive]
### Describe the Bug

As an admin user, I changed the name of the table 'page' to 'pages' on postgres. I refreshed directus app on my browser and the collection name is changed to 'pages' accordingly. But when I click on it, I get the error:

{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}

### To Reproduce

1. rename table 'page' on postgres to 'pages'
2. refresh directus app
3. click on Collections > pages

### What version of Directus are you using?

v9.0.0-rc.83

### What version of Node.js are you using?

v15.11.0

### What database are you using?

postgres 13.2

### What browser are you using?

brave

### What operating system are you using?

macOS

### How are you deploying Directus?

running locally

**Comment by oktasense** [comment]
The error goes away if I rename the postgres table back to 'page'

**Comment by rijkvanzanten** [comment]
Did you also rename all the internal references in the system tables? F.e. under directus_collections, directus_fields etc etc
